### PR TITLE
fix: wire up the step field and other migration fixes

### PR DESCRIPTION
**Pull request by elevatebart**
- Closes UNIFY-959

### Changes

- `module.export` -> `module.exports`
- i18n was sending a warning because `scope="global"` was missing from a few tags
- setup migration step in gql and its mutations
- change the label of the first step's button to reflect the skipping
- hide the last comment of `renameAuto` when skipping

### CODE REVIEW

File: packages/data-context/__snapshots__/config-generator.spec.ts.js [hunk 1]
```diff
@@ -1,72 +1,84 @@
...
 })
 `
 
 exports['migration utils should create a string for a config with global, component, and e2e options 1'] = `
 const { defineConfig } = require('cypress')
 
-module.export = defineConfig({
-  visualViewport: 300,
+module.exports = defineConfig({
+
```
Comment (ImCesar) [inline packages/data-context/__snapshots__/config-generator.spec.ts.js:39]: Is there anyway to trim this output?

**Review by lmiller1990 — DISMISSED** [review]

File: packages/data-context/__snapshots__/config-generator.spec.ts.js (hunk 1 above)
Comment (elevatebart) [reply packages/data-context/__snapshots__/config-generator.spec.ts.js]: Done in e5b166c

File: packages/data-context/src/actions/MigrationActions.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
+import { MIGRATION_STEPS } from '@packages/types'
 import type { DataContext } from '..'
 
+type MIGRATION_STEP = typeof MIGRATION_STEPS[number]
```
Comment (ImCesar) [inline packages/data-context/src/actions/MigrationActions.ts:4]: Is this still needed?

**Review by ImCesar — APPROVED** [review]
Looks good and works locally

**Pull request merged**